README.md

```diff
@@ -130,6 +130,7 @@ On the source node:
  * `cyedgehandles.start` : when starting to drag on the handle
  * `cyedgehandles.stop` : when the handle is released
  * `cyedgehandles.complete` : when the handle has been released and edges are created
+ * `cyedgehandles.cancel` : when the handle has been released but not on a valid target
```

Comment:
Maybe mention it uses the first `extraParams` for the rendered position? http://js.cytoscape.org/#cy.trigger

Otherwise, it's not clear how to get the data from the event callback.